lxd/storage/drivers/driver_btrfs_utils.go

```diff
@@ -384,3 +384,46 @@ func (d *btrfs) setSubvolumeReadonlyProperty(path string, readonly bool) error {
 	_, err := shared.RunCommand("btrfs", "property", "set", "-ts", path, "ro", fmt.Sprintf("%t", readonly))
 	return err
 }
+
+// BTRFSSubVolume is the structure used to store information about a subvolume.
+type BTRFSSubVolume struct {
```

Comment:
Do we actually use this externally? If not, then un-export.

Reply:
not in this PR but in the migration PR I use this struct to marshal json so thats why its exported.

Reply:
Oh, right, marshalling, makes sense